alpa/device_mesh.py

```diff
@@ -291,6 +299,13 @@ def init_collective_group(world_size, rank, backend, group_name):
                                   backend=backend,
                                   group_name=group_name)
 
+    @staticmethod
+    def init_broadcast_communicator(group_name, comm_key, world_size, device_ids, devices_global_rank, nccl_uid):
```

Comment:
Does this solve the previous to-do item (i.e., creating communicators in advance)?

Reply:
Yes, this part of code solves the to-do item,that indeed creates communicators in advance.